de-lib/src/main/java/org/iplantc/de/client/models/UserInfo.java

```diff
@@ -78,8 +80,41 @@ public void init(String userInfoJson) {
         preferences = userBootstrap.getPreferences();
         session = userBootstrap.getSession();
         appsInfo = userBootstrap.getAppsInfo();
-        workspace = appsInfo.getWorkspace();
-        systemIds = appsInfo.getSystemsIds();
+        if (appsInfo != null) {
```

Comment:
Well in the case where maybe the backend updates the key from `apps_info` to `app_info`, for example, we can avoid getting an NPE that would cause the UI to be stuck with an infinite loading spinner.